<?xml version="1.0" ?>

<launch>
  <arg name="robot_num" default="3" />
  <arg name="target" default="7" />
  

  <node name="give_goal_r3" pkg="franka_coord" type="give_goal_nr" output="screen">
    <param name="robot_num" value="$(arg robot_num)" />
    <param name="robot_id" value="3" />  
    <param name="speed_factor" value="0.05" /> 
    <param name="q_goal_num" value="$(arg target)" /> 
  </node>




</launch>
